cli/config.go

```diff
@@ -65,18 +115,29 @@ func loadConfig(cmd *cobra.Command) (Config, error) {
 	} else {
 		opts = append(opts,
 			config.WithPath("./"),
-			config.WithName("compass"),
+			config.WithName("compass.yaml"),
```

Comment:
@sudo-suhas the options `WithName("compass")` gives error `error in reading config` when the ./compass.yaml is missing.
Instead it shoould return the file not found error. Currently it is reading the compass executable and through the error. Can we rename the file to `.config` instead. And the line 118
`config.WithName("compass.yaml"),` will be changed to `config.WithName(".config"),`. Will remove the extension .yaml after that?